nemo/collections/asr/models/label_models.py

```diff
@@ -147,6 +156,12 @@ def training_step(self, batch, batch_nb):
 
         return {'loss': loss_value, 'log': tensorboard_logs, "n_correct_pred": n_correct_pred, "n_pred": len(labels)}
 
+    def training_epoch_end(self, outputs):
```

Comment:
Why log to stdout but not to wandb/tensorboard?

Reply:
Do we want double logging via logging.info as well as wandb and tensorboard ? This code runs on more than one process, so printed logs are bound to be out of order. For train logs, just maybe use the tensorboard/wandb logs.